cli/README.md

```diff
@@ -105,7 +123,8 @@ Sends the selected number of requests with a certain concurrency to provided URL
 * `--concurrency` (`-c`): Number of requests to make at the same time.
 * `--number` (`-n`): Total number of requests to send.
 * `--file` (`-f`): File with URLs (one URL per line) to run benchmark tests for.
-* `--output` (`-o`): The output format.
+* `--output` (`-o`): The output format: Either "table" or "csv".
```

Comment:
@felixarntz under all other "Arguments" headings I've noticed the usage of `--format` (`-f`) for the output format, but here it's defined as `--output` (`-o`). 

Is this correct? It may be correct in code (and function as `--output`) but should we be interchanging flags?

Reply:
@10upsimon I'm aware of this, this happened due to different people working on the commands and names diverging as such :)

I agree we should address this, but thinking better in a separate PR. Please feel free to open one!